inc_nx/nxu.h

```diff
@@ -860,15 +860,6 @@ typedef struct {
 #define INIT_CRC   0  /* crc32(0L, Z_NULL, 0) */
 #define INIT_ADLER 1  /* adler32(0L, Z_NULL, 0)  adler is initialized to 1 */
 
-/* prototypes */
-#ifdef NX_JOB_CALLBACK
-int nxu_run_job(nx_gzip_crb_cpb_t *c, void *handle,
-		int (*callback)(const void *));
-#else
-int nxu_run_job(nx_gzip_crb_cpb_t *c, void *handle);
-#endif
-
-
```

Comment:
File `samples/gzip_nxfht.c` needs to be modified too after this removal.

Reply:
@tuliom Done. I also fixed `samples/gzip_nxdht.c` and `samples/gunzip_nx_test.c`. All of them pull some parts from libnxz.a and also reimplement other parts. I tried to avoid increasing this dependency on libnxz.a but also didn't improve the situation much further. At least they will still compile and run as before.

In the future we should find a way to make these completely standalone to avoid such conflicts. But this is work for a separate PR.